draft-ietf-idr-linklocal-capability.xml

```diff
@@ -51,126 +51,113 @@
+      </t>
+      <t>
+        BGP speakers are now often deployed on point-to-point links in networks
+        where multihop reachability of any kind is not assumed or desired.  (All
+        next hops are assumed to be the reachable through a directly connected
+        point-to-point link.) This is common, for instance, in data center
+        fabrics <xref target="RFC7938"/>. In these situations, a Global IPv6
+        address is not required for the advertisement of reachability
+        information. In fact, providing Global IPv6 addresses in these kinds of
+        networks can be detrimental to Zero Touch Provisioning (ZTP).
+      </t>
+      <t>
+        Such BGP deployment models require BGP to run on each link, and any
+        simplification of BGP configuration can simplify orchestration and
+        configuration management. This proposal is a step in that direction.
+      </t>
+      <t>
+        With this new capability, the need for a Global unicast address assigned
         to the interfaces is eliminated.
       </t>
-      <t>Since IPv6 link-local addresses are not required to be globally unique,
-      implementations must ensure that they are strictly associated with a specific
-      interface.
+      <!-- XXX JMH - what is the intention of this next sentence?  That the
+      addresses are scoped? -->
+      <t>
+        Since IPv6 Link-Local addresses are not required to be globally unique,
+        implementations must ensure that they are strictly associated with a
+        specific interface.
       </t>
     </section>
     <!-- end of Introduction section -->
 
 <section numbered="true" toc="default">
       <name>Link-Local Next Hop Capability</name>
-      <t>The Link-Local Next Hop capability is a new BGP capability. A
-BGP speaker that supports capabilities advertisement <xref target="RFC5492"/>
-in an OPEN message should send this capability only when:
+      <t>
+        The Link-Local Next Hop capability is a new BGP capability. Its
+        Capability code is 77 and its Capability Length is 0.
       </t>
-      <ol spacing="normal" type="1"><li>It is capable of sending IPv6 link-local address as the only next
-              hop address for a route.</li>
-        <li>The implementation is capable of processing IPv6 link-local address
+      <t>
+        A BGP speaker that is willing to use (send and receive) IPv6
+        Link-Local-only next hops SHOULD advertise the Link-Local Next Hop
+        Capability to its peers only when:
+      </t>
+      <ol spacing="normal" type="1">
+        <li>It is capable of sending IPv6 Link-Local-only next hops for a route.</li>
+        <!-- XXX JMH - the text in this next sentence is awkward. Is the intent
+        to say that we're using the BGP peer configuration and that this
+        configuration is intended to be configured for a specific interface? -->
+        <li>The implementation is capable of processing IPv6 Link-Local address
```

Comment:
To translate this into a "technical/nerd" language => This is something when you have:

```
neighbor X::Y remote-as N
neighbor X::Y interface eth0
```
or (unnumbered)
```
neighbor eth0 remote-as N
```

But we could remove this sentence at all, IMO, if it adds just a confusion.

Reply:
Using the following:
        <li>IPv6 Link-Local neighbors are associated with interfaces as part of 
        their configuration to assist in determining the interface scope of
        received IPv6 Link-Local-only next hops.</li>